plone/stringinterp/interfaces.py

```diff
@@ -49,18 +48,18 @@ def substitutionList():
           [{'id':subId, 'description':subDescription}, ...]), ...  ]
         """
 
+
 class IContextWrapper(Interface):
     """
-        Wrap context in order to be able to provide custom strings
-        not stored on context
+    Wrap context in order to be able to provide custom strings
+    not stored on context
 
-        Usage:
+    Usage:
 
-        >>> wrapper = IContextWrapper(obj)(m1='A message', m2="Another one")
-        >>> notify(CustomEvent(wrapper))
+    &gt;&gt;&gt; wrapper = IContextWrapper(obj)(m1='A message', m2="Another one")
+    &gt;&gt;&gt; notify(CustomEvent(wrapper))
```

Comment:
Change this back.

```suggestion
    >>> wrapper = IContextWrapper(obj)(m1='A message', m2="Another one")
    >>> notify(CustomEvent(wrapper))
```

Reply:
probbaly some problem with zpretty and my "find" expression. ops.